src/components/Card/card.tsx

```diff
@@ -0,0 +1,42 @@
+import style from "./card.module.scss";
+import starOn from "../../../public/images/star-on.svg";
+import starOff from "../../../public/images/star-off.svg";
+import MoreSee from "../../../public/images/threeMoreDots.svg";
+import logo from "../../../public/images/logo.svg";
+import {useState} from "react";
+import Image from "next/image";
+
+function Card({}) {
```

Comment:
컴포넌트의 Props로 어떠한 값도 받아오고 있지 않기 때문에 객체는 지워줘야할 것 같아요~
```suggestion
function Card() {
```